### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Fahim-Khan-P**
### **I have added the following requirements**

- [ ] [ ] 1. Name field accepts up to 30 chars.
- [ ] [ ] 2. Email field accepts only emails.
- [ ] [ ] 3. Message field accepts up to 500 chars.
- [ ] [ ] 4. All fields have placeholder text.
- [ ] [ ] 5. All fields have the correct type.
- [ ] [ ] 6. A user cannot submit the form if one or more of the fields is empty.
- [ ] [ ] 7. Collect data by using the [Formspree service](https://formspree.io/html).
- [ ] [ ] 8. In order to lay out the elements on the page you should use Flexbox.
- [ ] [ ] 9. You don't need to implement any functionality that requires JavaScript.
- [ ] [ ] 10. You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- [ ] [ ] 11. Implement the button interactions (enable, hover, etc.)

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not pass all the linters
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,155 +1,236 @@
...
+        <input maxlength="30" class="form-elements" type="text" name="name" placeholder="Yeremias" required />
+        <input
+          class="form-elements"
+          type="email"
+          name="email"
+          placeholder="natajayanj14@gmail.com"
+          required
+        />
+        <textarea name="message" maxlength="500" class="form-elements" cols="50" rows="10" placeholder="Write your message here"></textarea>
+        <input type="submit" value="Get In Touch" id="btn-1"/>
```
Comment (1GORDON) [inline index.html]: Great job team 🤝

- [ ] Kindly implement all button interactions of `hover`, `pressed`, and  `enable` on the `Get In Touch` button.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Fahim-Khan-P & @Nuga0909 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions

- [ ] You may also update the MIT License link in your Readme file to make it more professional.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi Team , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay

## Optional suggestions
- [ ] Kindly update the MIT license path on the readme  file in order to have a professional readme 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**